### PR TITLE
Fix style removal when vendor.css is fingerprinted

### LINKED ISSUE #164: Running ember build twice w/o nombom in between causes fa-icons not to render
We run the below command and then promote `assets`, `fonts`, and the `index.html` to the backend application.  If I run the below command twice subsequently without blowing away and re-installing node modules, fa-icons are loaded; however, the element size is effectively 0x0 pixels.

`./node_modules/ember-cli/bin/ember build --env=production`

tried on both node 7.10 and 8.1.4 with same result.  On ember-font-awesome alpha (4.0.0-alpha.3).  Also tried on alpha.4 with same result

### PR DESCRIPTION
**Pull request by cibernox**
This might fix #164

### CODE REVIEW

**Comment by cibernox** [comment]
Do not merge this yet. I have reasons to think this might break the integrity of the vendor css

File: lib/prune-css.js [hunk 1]
```diff
@@ -56,7 +55,7 @@ module.exports = class PruneUnusedIcons extends BroccoliFilter {
   }
 
   getDestFilePath(relativePath) {
-    if (this.targetFiles.includes(relativePath)) {
+    if (/assets\/vendor(\-[\d|a-f]+)?\.css/.test(relativePath)) {
```
Comment (snewcomer) [inline lib/prune-css.js]: @cibernox I just ran this and it seems when the `assets/vendor.css` is run through this function, it doesn't have the fingerprint appended yet.  Second, do you want a `! not operator` here?

Comment (snewcomer) [reply lib/prune-css.js]: But then, will it process on first pass?

Comment (cibernox) [reply lib/prune-css.js]: That is what I'm investigating. I thought that fingerprinting would happen after this, but I found that I was receiving it fingerprinted. So, I believe I need to enforce execution order.

**Comment by cibernox** [comment]
I've released this and #166 in alpha.5.

I am not convinced if it's going to fix your problem, but it was certainly wrong.

**Comment by cibernox** [comment]
This hasn't fixed the bug, but at least now I'm able to reproduce it consistently.

**Pull request merged**